task/log.html

```diff
@@ -355,7 +347,9 @@ <h4 class="panel-title">
 }
 
 $(function () {
+    /*
```

Comment:
Is that a debugging leftover? I find it rather nice that the page self-updates.

Reply:
oh it is. It is annoying when you try to work with dom or css in browser and it reloads every few seconds. But surely it belong there in normal circumstances.